src/frontend/js/widgets/Dashboard/components/TeacherProfileDashboardSidebar/components/OrganizationThumb/index.tsx

```diff
@@ -0,0 +1,16 @@
+interface OrganizationThumbProps {
+  alt: string;
+  sizes: string;
+  src: string;
+  srcset: string;
+}
+
+const OrganizationThumb = ({ alt, sizes, src, srcset }: OrganizationThumbProps) => {
+  return (
+    <div className="organization-thumb">
+      <img alt={alt} sizes={sizes} src={src} srcSet={srcset} />
+    </div>
```

Comment:
I think the `div` is useless here. We could use `object-position` and `object-fit` to manage image display according its dimensions.

Reply:
I did it with the first solution that i had in mind but i'll look at using these props.